client/app/scripts/superdesk-desks/desks.js

```diff
@@ -368,6 +368,26 @@
         };
     }
 
+    SluglinesItemListDirective.$inject = ['api'];
+    function SluglinesItemListDirective(api) {
+        return {
+            templateUrl: 'scripts/superdesk-desks/views/slugline-items.html',
+            scope: {
+                desk: '='
+            },
+            link: function(scope, elem) {
+                scope.items = [];
+                scope.loading = true;
+                api.get('desks/' + scope.desk + '/sluglines').then(function(items) {
+                    scope.items = items._items;
+                    scope.loading = false;
+                }, function() {
+                    scope.loading = false;
+                });
```

Comment:
The `scope.loading` flag can be cleared in `finally()`, avoiding repetition. The error handler can then be dropped as well.

(although it's relatively minor since the logic is so simple)